src/In.ProjectEKA.HipService/DataFlow/DataFlow.cs

```diff
@@ -53,6 +55,55 @@ public class DataFlow : IDataFlow
             return new Tuple<HealthInformationResponse, ErrorRepresentation>(response, errorRepresentation);
         }
 
+        private static Tuple<HealthInformationResponse, ErrorRepresentation> ConsentArtefactNotFound()
+        {
+            return new Tuple<HealthInformationResponse, ErrorRepresentation>(null,
+                new ErrorRepresentation(new Error(ErrorCode.ContextArtefactIdNotFound,
+                    ErrorMessage.ContextArtefactIdNotFound)));
+        }
+
+        public async Task<Tuple<LinkDataResponse, ErrorRepresentation>> HealthInformationFor(
+            string linkId,
+            string token,
+            string transactionId)
+        {
+            var linkData = await linkDataRepository.GetAsync(linkId);
+            if (linkData == null) return LinkDataNotFound();
+            if (linkData.Token != token) return InvalidToken();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5067092)